src/core/ext/filters/client_channel/lb_policy/xds/xds.cc

```diff
@@ -273,7 +273,7 @@ class XdsLb : public LoadBalancingPolicy {
   bool PickFromRoundRobinPolicyLocked(bool force_async, PendingPick* pp,
```

Comment:
Please rename all methods that include the string `RoundRobin` to instead use `Child`.